### ISSUE TITLE
openfmri: crawl derivatives into submodule(s)

### ISSUE THREAD

**Issue by yarikoptic** [datalad/datalad#703]
ATM all BIDS ones got already derivatives/mriqc and some got even others.  Some of them are really heavy on number of files:
```
$> du -sc --inodes ds*/derivatives/*
45	ds000007/derivatives/mriqc
246	ds000009/derivatives/mriqc
2820	ds000030/derivatives/event_plots
1	ds000030/derivatives/Final_Scan_Count.pdf
48	ds000030/derivatives/mriqcp
39	ds000030/derivatives/parameter_plots
1123	ds000030/derivatives/physio_plots
31	ds000052/derivatives/mriqc
47	ds000101/derivatives/mriqc
57	ds000102/derivatives/mriqc
76	ds000119/derivatives/mriqc
57	ds000120/derivatives/mriqc
61	ds000121/derivatives/mriqc
20	ds000122/derivatives/mriqc
448	ds000133/derivatives/mriqc
65	ds000157/derivatives/mriqc
111	ds000168/derivatives/mriqc
35	ds000170/derivatives/mriqc
45	ds000174/derivatives/mriqc
39	ds000177/derivatives/mriqc
567	ds000201/derivatives/event_plots
256	ds000201/derivatives/high-resolution_mosaics
334	ds000201/derivatives/mriqc
10130	ds000203/derivatives/mriqc
16701	total
```

ATM they all seems to ship derivatives in a single tarball, but I guess that situation would change soon whenever more derivatives would come about and it would become not sensible to ship all of them in the same tarball.
 @chrisfilo , @poldrack -- any plans for splitting them consistently, into e.g. `ds<ID>_R<VERSION>_derivative_<DERIVATIVE_ID>.tgz` ?

Then it would make sense for us to provide a submodule per each derivative...

**Comment by chrisfilo** [comment]
No. But will have s3 access to individual files for all datasets soon. This
should solve the problem.

**Comment by yarikoptic** [comment]
That is great, but what of the following will apply @chrisfilo 
1.  provide urls to each derivative file on S3 from openfmri.org
2. will  keep growing 1 huge tarball for all derivatives
3. will stop shipping tarballs with derivatives altogether
?

**Comment by chrisfilo** [comment]
#1 will happen for sure. #3 will not happen. In terms of #2 we will try to
split tarballs of future derivatives into more manageable size.

**Comment by yarikoptic** [comment]
so, regarding #2 -- the answer should have been "yes", not "no" ;)
the question now is how would you split them?  I was suggesting to split them per each derivative analysis and name consistently, i.e. `ds<ID>_R<VERSION>_derivative-<DERIVATIVE_ID>.tgz`  which Mr.BIDS should have loved! ;)

**Comment by chrisfilo** [comment]
Yes. Sorry I misread your message. That makes a lot of sense. The BIDS
derivatives spec is slowly moving towards one big directory per pipeline
which fits this scheme very well.

**Comment by mih** [comment]
Related #1671
